### DISCUSSION TITLE
NPM supports isolated node_modules?

### DISCUSSION THREAD

**Discussion by oravecz** [pnpm/pnpm#4447] [Q&A]
The [feature comparison chart](https://pnpm.io/feature-comparison) indicates that `npm` supports isolated node_modules? I thought this was in an RFC stage, but not yet available?

**Comment by zkochan** [comment]
You are correct. It is not yet available. It is a mistake in the docs.

I have pushed a fix https://github.com/pnpm/pnpm.github.io/commit/73ff31f9896f2167a9e661084ac4197cd1c53280
